src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -71,7 +71,13 @@ protected override void DisconnectHandler(MapView platformView)
 
 		protected override MapView CreatePlatformView()
 		{
-			MapView mapView = new MapView(Context);
+			GoogleMapOptions options = new GoogleMapOptions();
+			
+			string? mapId = GetMapId();
+			if (mapId != null)
+				options.InvokeMapId(mapId);
+  
+			MapView mapView = new MapView(Context, options);
 			mapView.OnCreate(s_bundle);
 			mapView.OnResume();
```

Comment:
Should these be moved into the ConnectHandler code so that we only resume once the handler is actually in use, allowing for a better/easier means to override the `CreatePlatformView` of the use if `PlatformViewFactory`?

Regardless, I would think that this really does fall into the category of a platform-specific that can be set per-map view?